<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: qconicalgradient(cx:0.438, cy:0.591, angle:35.1, stop:0 rgba(75, 162, 168, 255), stop:1 rgba(255, 255, 255, 255));
	border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>100</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #ff74c7;
	border:none;
}</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>94</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #ff74c7;
}</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>230</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #ff74c7;
}</string>
       </property>
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	
	background-color: rgb(225, 213, 255);
	color: rgb(10, 9, 21);
	border: 2px solid rgb(251, 138, 255);
	border-radius: 5px;
	
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>170</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	
	background-color: rgb(225, 213, 255);
	color: rgb(10, 9, 21);
	border: 2px solid rgb(251, 138, 255);
	border-radius: 5px;
	
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_amount">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>230</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	
	background-color: rgb(225, 213, 255);
	color: rgb(10, 9, 21);
	border: 2px solid rgb(251, 138, 255);
	border-radius: 5px;
	
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter amount</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Submit_pushButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>310</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #cdcdff;
	border-radius: 10px;
	color: #ff74c7;
}

QPushButton:hover{
	background-color: #d439ff;
	border-radius: 10px;
	border-bottom: 3px solid #550000;
	color: black;
}


QPushButton:pressed{
	background-color: #7777fd;
	border-radius: 10px;
	border-top: 3px solid #fff263;
	color: #aa00ff;
}

</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>531</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: #7777ff;
}
</string>
       </property>
       <property name="text">
        <string>BANK ACCOUNT MANAGEMENT SYSTEM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
